trace/cloud-trace-demo-app/contributing.md

```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```

Comment:
This is no longer needed because of the contributing.md in the root directory

Reply:
Removed.